Projects/RevenueCatUI/Project.swift

```diff
@@ -0,0 +1,75 @@
+import ProjectDescription
+import ProjectDescriptionHelpers
+
+let allDestinations: Destinations = [
+    .iPhone,
+    .iPad,
+    .mac,
+    .macWithiPadDesign,
+    .macCatalyst,
+    .appleWatch,
+    .appleTv,
+    .appleVision,
+    .appleVisionWithiPadDesign
+]
+
+let allDeploymentTargets: DeploymentTargets = .multiplatform(
+    iOS: "15.0",
+    macOS: "11.0",
+    watchOS: "7.0",
+    tvOS: "14.0"
+)
+
+let project = Project(
+    name: "RevenueCatUI",
+    organizationName: "RevenueCat, Inc.",
+    targets: [
+        .target(
+            name: "RevenueCatUI",
+            destinations: allDestinations,
+            product: .framework,
+            bundleId: "com.revenuecat.RevenueCatUI",
+            deploymentTargets: .iOS("15.0"),
```

Comment:
Also not using deployment targets defined above, which also seem to have different versions in the current project.